distributed/protocol/tests/test_pandas.py

```diff
@@ -32,13 +33,16 @@
     pd.DataFrame({'x': [b'a', b'b', b'c']}),
     pd.DataFrame({'x': pd.Categorical(['a', 'b', 'a'], ordered=True)}),
     pd.DataFrame({'x': pd.Categorical(['a', 'b', 'a'], ordered=False)}),
+    pd.Series(np.arange(10000000)),
```

Comment:
should test various dtypes (int16, float32), datetimes, timedelta, period (``pd.period_range``), datetime w/tz (pd.date_range(...., tz='US/Eastern')